src/components/SelectList/SelectList.tsx

```diff
@@ -309,21 +312,48 @@ const MultiValueRemove = props => (
     </ReactSelectComponents.MultiValueRemove>
 );
 
+const LightSchemeMenu = (children, props: WithSelectProps<MenuProps>) => (
+    <LightScheme>
+        <ReactSelectComponents.Menu {...props}>{children}</ReactSelectComponents.Menu>
+    </LightScheme>
+);
+
```

Comment:
Is there a need to have children as a first argument? I believe it should be a regular prop 🤔 

```suggestion
const LightSchemeMenu = (props: WithSelectProps<MenuProps>) => (
    <LightScheme>
        <ReactSelectComponents.Menu {...props}>{props.children}</ReactSelectComponents.Menu>
    </LightScheme>
);

```